Doc/library/asyncio-policy.rst

```diff
@@ -222,6 +222,9 @@ implementation used by the asyncio event loop:
       This method has to be called to ensure that underlying
       resources are cleaned-up.
 
+   .. deprecated:: 3.12
```

Comment:
(How sure are you that this placement will make it apply to the whole class instead of just to `close()`?)

Reply:
<img width="423" alt="image" src="https://user-images.githubusercontent.com/59607654/201462038-98585631-94e2-4d3e-bd24-f7ba0184517c.png">

It applies to the whole class.